plugins/org.jboss.tools.livereload.core/src/org/jboss/tools/livereload/core/internal/util/WSTUtils.java

```diff
@@ -74,6 +73,7 @@
 	public static final String JBOSSAS_SERVER_PORT = "org.jboss.ide.as.serverport";
 	public static final String TOMCAT_60_SERVER_TYPE = "org.eclipse.jst.server.tomcat.60";
 	public static final String TOMCAT_70_SERVER_TYPE = "org.eclipse.jst.server.tomcat.70";
+	public static final String EJB_MODULE_TYPE = "jst.ejb";
```

Comment:
Is this constant available somewhere in JST for us to reuse? If yes, we should reuse the original constant instead.